### ISSUE TITLE
High contrast issues

### ISSUE THREAD

**Issue by alon24** [carbon-design-system/carbon#7244]
Based on https://github.ibm.com/security-services/secrets-manager-project-management/issues/949

Element do not inherit the selected high contrast theme
turn on high contrast mode（with high contrast chrome extention）

 1. data table the pages are not high contrast 
![image](https://user-images.githubusercontent.com/5415443/98463862-3ff28900-21c7-11eb-82ad-ff1f5d582d20.png)

2, Edit TextInput
Try to fill or edit the value, the auto-completion dropdown list is not shown as high contrast theme
![image](https://user-images.githubusercontent.com/5415443/98463888-644e6580-21c7-11eb-91e1-70af0e3a2594.png)

**Comment by tw15egan** [comment]
These are both browser-based elements, not Carbon elements. We don't do any styling on select menus/input auto-complete menus, so I recommend overriding on your end if you'd like these to match. 

We try to avoid any styling to browser/os specific elements due to the inherent variability across browsers / OS's

**Comment by alon24** [comment]
based on this input and https://ibm-cloudplatform.slack.com/archives/C036P1CTN/p1604992574372400 I will close this issue
